website_blog_excerpt_img/i18n/es.po

```diff
@@ -0,0 +1,38 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* website_blog_excerpt_img
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-02-11 12:37+0000\n"
+"PO-Revision-Date: 2016-02-11 13:38+0100\n"
+"Last-Translator: Jairo Llopis <j.llopis@grupoesoc.es>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"Language: es\n"
+"X-Generator: Poedit 1.8.6\n"
+
+#. module: website_blog_excerpt_img
+#: view:website:website_blog.blog_post_short
+msgid "Read more"
+msgstr "Leer más"
+
+#. module: website_blog_excerpt_img
+#: view:website:website_blog.blog_post_short
+msgid "col-xs-10"
```

Comment:
Yes, those are Bootstrap classes that get hit by the translator, but should not be modified.